app/controllers/organisations_controller.rb

```diff
@@ -4,7 +4,7 @@ class OrganisationsController < ApplicationController
   before_action :set_organisation, only: %i[show]
 
   def show
-    @vacancies = @organisation.vacancies.live
+    @vacancies = @organisation.all_vacancies.live
```

Comment:
Are we getting any ordering here? The ticket requires a particular order when listing them (Trust jobs first, schools jobs after)

Reply:
thanks - came up with a nice plan in the view